src/wallet/api/wallet2_api.h

```diff
@@ -111,6 +110,13 @@ struct PendingTransaction
     virtual std::vector<std::string> signersKeys() const = 0;
 };
 
+struct StakeUnlockResult
+{
+  virtual bool success() = 0;
+  virtual std::string msg() = 0;
+  virtual PendingTransaction * ptx() = 0;
```

Comment:
Formatting: spaces on *both* sides of the `*` just hurts.  (I realize this was copied from the existing code style, but that really just means the other code needs to be fixed).

---

Aside: there's an age-old debate about whether the space belongs on the left or right of the `*` (and `&` for references).  C programmers strongly prefer `int *var; int &bar;` while C++ programmers are more mixed, but tend to prefer `int* var; int& bar` (unless they started as C programmers).

Bjarne Stroustrap's answer on this was pretty convincing to me: https://www.stroustrup.com/bs_faq2.html#whitespace

Before I read it, I was in the `int *var; int &bar;` camp; after I read it I completely switched and now use `int* var; int& bar` in all new code.

But all that said: I can easily deal with either, but spaces on *both* sides (or neither side) just hurts my eyes too much.  In this case either: `virtual PendingTransaction* ptx() = 0;` or `virtual PendingTransactions *ptx() = 0;`